juriscraper/pacer/email.py

```diff
@@ -351,20 +353,17 @@ def _contains_attachments_plain(self) -> bool:
 
         :returns: True if it contains otherwise False.
         """
-
         mail_body = self.tree.text_content()
         regex = r"^.*?The following document\(s\) are associated with this transaction:(.*?)$"
-
```

Comment:
I noticed that in the new plain_text example, the list of attachment documents is not the last section of the notification.

Would it be a good idea to tweak the regex so that it only captures the lines likely to be part of the attachments section, allowing the parsing to finish earlier? Something like in the description parsing:

`regex = r"^.*?The following document\(s\) are associated with this transaction:(?P<attachments>.*?)(electronically mailed to:|$)"`

Reply:
Great suggestion I have included this in the PR